From 984ca669d090dcf9cf0a018b3f48cb3d1eb50e7e Mon Sep 17 00:00:00 2001
From: acho98 <sdfgh98@naver.com>
Date: Wed, 15 Dec 2021 05:34:20 +0900
Subject: [PATCH] ADD tiger

---
 tiger | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 tiger

diff --git a/tiger b/tiger
new file mode 100644
index 0000000..c1851a2
--- /dev/null
+++ b/tiger
@@ -0,0 +1 @@
+name: bobo
\ No newline at end of file
-- 
2.34.1.windows.1

